test-swift-docc/test-swift-docc.txt

```diff
@@ -1,8 +1,17 @@
-// Check Swift-DocC can compile and preview documentation.
-//
-// RNU: rm -rf %S/Output
-// RUN: %{docc} convert %S/DocCTest/Sources/DocCTest/DocCTest.docc --output-path %S/Output
-// RUN: ls %S/Output
-
-// REQUIRES: rdar91638258
-
+// Check that Swift-DocC can compile documentation.
+//
+// Clear any existing state:
+// RUN: rm -rf %t.dir
```

Comment:
@bnbarham pointed out that it makes more sense to use `%t`, the allotted testing directory, for our temporary output instead of %S.